### PR TITLE
Added notes for better understanding of what needs to be done :D

### PR DESCRIPTION
**Pull request by Beta-King**
http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by Beta-King** [comment]
Solves #506 :D

File: README.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ Labyrinth
 - [**Hints for GCI students**](#hints-for-gci-students)
 - [**Solve an Issue**](#solve-an-issue)
 - [**UI identity guideline**](#ui-identity-guideline)
-- [**Videos about the Labyrinth**](#videos-by-students)
 - [**Maintainers**](#maintainers)
+- [**Maintainers**](#videos-by-students)
```
Comment (niccokunzmann) [inline README.md]: Would you like to change this? This reverts #460

File: README.md [hunk 2]
```diff
@@ -94,6 +94,8 @@ Tiles are present in the `tiles` folder within subdirectories corresponding to p
 To create a tile you may use an svg editor such as inkscape. However other photo editors and formats do wok if they are imported into the editor
 and saved as a svg file with the speccified dimensions.
 
+Note: if you are copying the template of a tile(floor) from an existing tile, then do not edit it as a png but directly as a svg. This is so that errors in alignment do not exist and the tile(floor) is perfectly aligned.
```
Comment (niccokunzmann) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -163,6 +165,8 @@ The license numbers are as follows :
 
 <br><br>
 
+Note: if you are adding music to a tile then add it to: `audio/music` and if you are adding music as a background then add it to: `audio/background`! This is so that we can be organised and have a consistent filing system.
```
Comment (niccokunzmann) [inline README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -313,7 +317,6 @@ The FOSSASIA Labyrinth allows you to contribute parts to a huge labyrinth. Pleas
 - https://www.youtube.com/watch?v=XvoHY3QCJtw
 - https://www.youtube.com/watch?v=0Z144cuITCE
 - [Labyrinth promotional video by Supun Tharinda Edirisuriya](https://www.youtube.com/watch?v=tHeScGOkJ3M)
-- [Labyrinth - The Maze Game | Promo by Naveen Rajan](https://www.youtube.com/watch?v=R8dZjl7qqY8)
```
Comment (niccokunzmann) [inline README.md]: I think this might be an unintended removal. What is your opinion?

**Comment by niccokunzmann** [comment]
@Beta-King I commented. What do you think?

File: README.md (hunk 4 above)
Comment (Beta-King) [reply README.md]: yup unintended :D changing that :D

**Comment by Beta-King** [comment]
It was unintended, wait ill change it :D

**Comment by Beta-King** [comment]
@niccokunzmann made changes :D please merge !!

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**